ydb/core/tx/schemeshard/schemeshard__login.cpp

```diff
@@ -153,6 +154,9 @@ struct TSchemeShard::TTxLogin : TTransactionBase<TSchemeShard> {
                 Result->Record.SetError(TStringBuilder() << "User " << loginRequest.User << " is locked out");
                 return true;
             }
+        } else if (!isEnabled) {
```

Comment:
May be move this check to function `CheckAccountLockout()`. `isEnabled` check that user is locked out